internal/provider/resource_cloudexport.go

```diff
@@ -29,11 +32,16 @@ func resourceCloudExportCreate(ctx context.Context, d *schema.ResourceData, m in
 
 	req := *cloudexport.NewV202101beta1CreateCloudExportRequest()
 	req.Export = export
-
+	jsonReq, err := json.Marshal(*export)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	tflog.Debug(ctx, fmt.Sprintf("Kentik API request - create:\n%s\n", jsonReq))
```

Comment:
```suggestion
	tflog.Debug(ctx, "Create cloud export Kentik API request"request", req)
```
I am not convinced about encoding data to JSON, because it might distort information about the object, that might be useful for debugging (because of `omitempty`, custom marshallers, etc.). If `req` is not enough consider using fmt.Sprintf("%+v", req) to display names of struct fields.

Also, failing to prepare data only for logging **should not** result in failure of the provider.

Reply:
Done